lib/faker/uuid.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Faker.UUID do
+  @moduledoc """
+  Functions for generating UUID's.
+  """
+
+  @uuid_v4 4
+
+  @variant10 2
+
+  @doc """
+  Generate a random v4 UUID.
+  """
+  @spec v4() :: String.t
+  def v4 do
+    <<u0::48, _::4, u1::12, _::2, u2::62>> = :crypto.strong_rand_bytes(16)
```

Comment:
Hi, sorry for late response. Yes, we have faker specific random for bytes, you can use `Faker.random_bytes/1`. I'll merge it and change to not delay this MR. Thank you for your contribution!